src/main/java/uk/gov/hmcts/probate/service/ocr/OCRToCCDMandatoryField.java

```diff
@@ -13,19 +14,32 @@
 @Service
 public class OCRToCCDMandatoryField {
 
-    public List<String> ocrToCCDMandatoryFields(List<OCRField> ocrFields) {
+    public List<String> ocrToCCDMandatoryFields(List<OCRField> ocrFields, FormType formType) {
         List<String> descriptions = new ArrayList<>();
         List<String> ocrFieldNames = new ArrayList<>();
         ocrFields.forEach(ocrField -> {
             ocrFieldNames.add(ocrField.getName());
         });
-        Stream.of(MandatoryFields.values()).forEach(field -> {
-            log.info("Checking {} against ocr fields", field.getKey());
-            if (!ocrFieldNames.contains(field.getKey())) {
-                log.warn("{} was not found in ocr fields", field.getKey());
-                descriptions.add(field.getValue());
-            }
-        });
+        switch (formType) {
+            case PA8A:
+                Stream.of(CaveatMandatoryFields.values()).forEach(field -> {
+                    log.info("Checking {} against ocr fields", field.getKey());
+                    if (!ocrFieldNames.contains(field.getKey())) {
+                        log.warn("{} was not found in ocr fields", field.getKey());
+                        descriptions.add(field.getValue());
+                    }
+                });
+                break;
+            default:
```

Comment:
Should this be explicit and maybe default captures an error state?

Reply:
no

Reply:
at this point the errors are caught already, please see controller and enum